static-site/FAQ.md

```diff
@@ -36,46 +34,61 @@ permalink: /faq.html
+BFD receives data from two different data source maintainers (CCW- Chronic Conditions Warehouse & RDA- Replicated Data Access API). 
+
+Weekly, on Friday afternoons, BFD receives beneficiary data and adjudicated claims data supporting the FHIR Coverage, ExplanationOfBenefit, and Patient resources via weekly RIF files from CCW. The data is loaded into BFD no later than 9:00 AM ET the next business day (typically Monday).
 
-BFD also receives beneficiary data and adjudicated claims data weekly via RIF files from CCW on Friday afternoons.
-The data is loaded into BFD no later than 9:00 AM ET the next business day (typically Monday). If there is any delay past this time frame, BFD will declare an incident and notify its partners.
+Partially adjudicated claims data supporting the FHIR Claim and ClaimResponse resources is updated nightly. This data is provided via RDA.
 
-#### What is the difference between V1 (Version 1) and V2 (Version 2) of the BFD API?
+Please note that The amount of time it takes a claim to progress through the data ecosystem depends on a variety of factors from the data source maintainers. A CMS [whitepaper](https://confluence.cms.gov/display/BFD/FHIR+and+CMS+Documentation+Links?preview=/415007408/455427981/medicare-claims-maturity%20(1).pdf) documents some additional details on the claim lifecycle.
 
-V1 of the BFD API works towards conformance to [STU3](https://www.hl7.org/fhir/STU3/), while V2 of the BFD API is conformant to FHIR [R4](https://hl7.org/fhir/R4/). 
+#### What is the difference between Version 1 (v1) and Version 2 (v2) of the BFD API?
 
-Another key difference between V1 and V2 is that partially adjudicated claim data is only available in V2. 
+BFD's v1 of the API stems from the STU3 FHIR specification, while v2 of the BFD API stems from the R4 FHIR Specification
 
-Additional details about more technical differences can be found on this [page](https://github.com/CMSgov/beneficiary-fhir-data/wiki/Migrating-to-V2-FAQ) which also highlights how to upgrade from V1 to V2.
-Lastly, there are is no deadline to deprecate V1. However, there is no active feature development.
+Additional details about more technical differences can be found on this [page](https://github.com/CMSgov/beneficiary-fhir-data/wiki/Migrating-to-V2-FAQ) which also highlights how to upgrade from v1 to v2. While there is no published deadline to deprecate v1 new features are not being added to v1.
+
+Currently, all of the Peering Partner APIs still utilize v1 for some clients with DPC solely using v1. All Peering Partners are highly recommended to make plans to move to v2. 
 
 #### Where can I find the BFD data dictionary?
-The data dictionary can be accessed [here](/data-dictionary-v1) for V1 and [here](/data-dictionary-v2) for V2.
-The data dictionary is updated in each release if there are changes.
+The data dictionary can be accessed [here](https://d2tcxctwdgn0ff.cloudfront.net/index.html).
```

Comment:
I think we should probably leave the data dictionary links as they were. The relative links (/data-dictionary-v1 and /data-dictionary-v2) will resolve correctly when the site is deployed.